tests/pthread/test_pthread_mutex.cpp

```diff
@@ -107,7 +108,7 @@ int main()
 		for(int i = 0; i < NUM_THREADS; ++i)
 			CreateThread(i, threadNum++);
 
-		emscripten_set_main_loop(WaitToJoin, 0, 0);
+		emscripten_set_timeout_loop(WaitToJoin, 100, 0);
 	} else {
```

Comment:
same question for this file - why doesn't it work in minimal_runtime without these?

Reply:
MINIMAL_RUNTIME does not support library_browser.js because the $Browser object does not DCE well if linked in. This test was not looking to exercise `emscripten_set_main_loop`, so changed to `emscripten_set_timeout_loop` for the same effect.